src/rust/ide/lib/ast/impl/src/lib.rs

```diff
@@ -311,8 +311,8 @@ impl Ast {
         self.iter_recursive().find(|ast| ast.repr() == repr)
     }
 
-    /// Get the index (relative to self) for given child node.
-    pub fn index_of_child(&self, child:&Ast) -> FallibleResult<Index> {
+    /// Get the offset relative to self for a given child node.
```

Comment:
I would still extend this docs. To be more clear that this is an offest of "begins of text representation" in number of codepoints (although the last can be deduced from the returned type).

Reply:
I've extended somehow, though didn't feel like mentioning codepoints. The type expresses it, we should not need to repeat it on every usage.